routes/cache.js

```diff
@@ -51,6 +61,27 @@ router.get('/item', function (req, res) {
   return null
 })
 
+// GET multiple cache items
+router.post('/getitems', function (req, res) {
+  const keys = JSON.parse(req.body)
+  const memcached = req.app.locals.memcached
+  if(!Array.isArray(keys)) {
+    return res.status(400).json(jerr.BadRequest)
+  }
+
+  memcached.getMulti(keys, function (err, data) {
+    if (err) {
+      res.status(500).json(jerr.InternalServer)
+    } else {
+      Object.keys(data).forEach(function(k) {
+        data[k] = data[k].toString()
+      })
+      res.json(data)
+    }
+  })
+  return null
```

Comment:
This is not needed. The return value of the middleware functions are just ignored.

If it was added so the function consistently return something, line 69 should have been in two: send the response, then `return`.

Reply:
Done!